docs/entitlements-flow.md

```diff
@@ -87,6 +87,24 @@ subscriptions.setOnEntitlementsResponse(function(entitlementsPromise) {
   });
 })
 ```
+## Validate entitlements
+To verify the entitlements from Google are valid and authorised, check the validity of the JWT representation of the entitlements.
+
+The following registered claims are used:
+
+| Claim | Value | Description |
+| ----- | ----- | ----------- |
+| `exp` | [NumericDate](https://tools.ietf.org/html/rfc7519#page-6) (e.g. `1543596936` ) | When the token expires |
+| `iat` | [NumericDate](https://tools.ietf.org/html/rfc7519#page-6) (e.g. `1543595136` ) | When the token was issued |
+| `iss` | `subscribewithgoogle@system.gserviceaccount.com` | Issuer of the token |
+| `aud` | URL (e.g. `https://example.com` ) | Audience of the token which will be the publisher's URL |
+
+To validate the signature, use the keys found here:
+https://www.googleapis.com/robot/v1/metadata/x509/subscribewithgoogle@system.gserviceaccount.com
```

Comment:
Please use `[]()` link markup. And let's try to combine these two into a single sentence. E.g. "[x509 format] and [JWK format]".

Reply:
Done.